doc/neps/roadmap.rst

```diff
@@ -42,53 +53,41 @@ improve the dtype system.
   - One of these should probably be the default for text data. The current
     behavior on Python 3 is neither efficient nor user friendly.
 
-- ``np.dtype(int)`` should not be platform dependent
-- Better coercion for string + number
 
 Performance
 -----------
 
 We want to further improve NumPy's performance, through:
 
-- Better use of SIMD instructions, also on platforms other than x86.
+- Better use of SIMD instructions, also on platforms other than x86 - see `NEP 38`_.
 - Reducing ufunc overhead.
 - Optimizations in individual functions.
 
 Furthermore we would like to improve the benchmarking system, in terms of coverage,
 easy of use, and publication of the results (now
 `here <https://pv.github.io/numpy-bench>`__) as part of the docs or website.
 
+
 Website and documentation
 -------------------------
 
-Our website (https://numpy.org) is in very poor shape and needs to be rewritten
-completely.
+Our website (https://numpy.org) was completely redesigned recently. We aim to
+further improve it by adding translations, better Hugo-Sphinx integration via a
+new Sphinx theme, and more (see `this tracking issue <https://github.com/numpy/numpy.org/issues/266>`__).
 
-The NumPy `documentation <https://www.numpy.org/devdocs/user/index.html>`__ is
-of varying quality - in particular the User Guide needs major improvements.
+The NumPy `documentation <https://www.numpy.org/devdocs>`__ is of varying
+quality. The API documentation is in good shape, high-level documentation on
+many topics is missing or outdated. See `NEP 44`_ for planned improvements.
 
-Random number generation policy & rewrite
------------------------------------------
 
-A new random number generation framework with higher performance generators is
-close to completion, see `NEP 19`_ and `PR 13163`_.
-
-Indexing
---------
-
-We intend to add new indexing modes for "vectorized indexing" and "outer indexing",
-see `NEP 21`_.
-
-Continuous Integration
-----------------------
+Type annotations
+----------------
 
-We depend on CI to discover problems as we continue to develop NumPy before the
-code reaches downstream users.
+We aim to add type annotations for all NumPy functionality, so users can use
+tools like `mypy`_ to type check their code and IDEs can improve their support
+for NumPy. The existing type stubs in the `numpy-stubs`_ repository are being
+improved and will be moved into the NumPy code base.
 
-- CI for more exotic platforms (if available as a service).
-- Multi-package testing
-- Add an official channel for numpy dev builds for CI usage by other projects so
-  they may confirm new builds do not break their package.
 
 Other functionality
```

Comment:
Maybe this could be `Maintenance` I think that captures the current bullet points better, or maybe keep something more general, and just use it for all more concrete proposals.

Reply:
Anyway, I am good with putting this in. I liked the more top-level category idea at some point, I think you used it in the SciPy talk?

Reply:
Good point, changing to `Maintenance`.

And yes, in my PyData Amsterdam talk:

![image](https://user-images.githubusercontent.com/98330/83336480-14807f80-a2b4-11ea-9bbb-247479fe3837.png)